ssl/ssl_cert.c

```diff
@@ -1020,6 +1020,8 @@ static int ssl_security_default_callback(const SSL *s, const SSL_CTX *ctx,
     case SSL_SECOP_CIPHER_CHECK:
         {
             const SSL_CIPHER *c = other;
+            int pfs_mask = SSL_kEDH | SSL_kEECDH | SSL_kECDHEPSK | SSL_kDHEPSK;
```

Comment:
This PR could be a good candidate to also consistently use `SSL_kDHE` instead of `SSL_kEDH` in our codebase, and the same goes for `SSL_kECDHE` over `SSL_kEECDH`.
We already have them defined as respective aliases, and even in the OTC we were not entirely sure about the meaning of the less usual aliases used here.

https://github.com/openssl/openssl/blob/a044af49c43ec8fe099deeb5d06501ddf70abf7a/ssl/ssl_local.h#L165-L172